tests/unit/workers/AnalyzeToxicityWorker.test.js

```
@@ -2630,4 +2630,9 @@ describe('AnalyzeToxicityWorker', () => {
       }
     });
   });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+    jest.clearAllMocks();
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow afterEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)